zio-http/src/test/scala/zio/http/internal/HttpGen.scala

```diff
@@ -267,4 +267,13 @@ object HttpGen {
 
   def allowHeader: Gen[Any, Allow] =
     Gen.chunkOfBounded(1, 9)(allowHeaderSingleValue).map(Allow.AllowMethods.apply)
+
+  def connectionHeader: Gen[Any, Connection] =
+    Gen.fromIterable(
```

Comment:
This will create a deterministic generator of these three values. I think we want a random generator instead so we should use `Gen.elements` here.

Reply:
Got it.  It is changed to elements.

Reply:
Doesn't look like it is changed.